deeprank2/neuralnets/gnn/naive_gnn.py

```diff
@@ -7,7 +7,14 @@
 # ruff: noqa: ANN001, ANN201
 
 
-class NaiveConvolutionalLayer(nn.Module):  # noqa: D101
+class NaiveConvolutionalLayer(nn.Module):
+    """Naive convolutional layer for graph neural networks.
```

Comment:
unclear what Naive means in this context (unless it's a known term in ML/DL?)

Reply:
I renamed it to VanillaNetwork and VanillaConvolutionalLayer, and changed naive to vanilla everywhere. That is a know term in ML/DL